shared/templates/sebool/ansible.template

```diff
@@ -4,34 +4,44 @@
 # complexity = low
 # disruption = low
 
+{{% if product in ['ol8', 'ol9', 'rhel8', 'rhel9'] %}}
+{{% set PACKAGE_NAME = "python3-libsemanage" %}}
+{{% elif product == "sle15" %}}
+{{% set PACKAGE_NAME = "policycoreutils" %}}
+{{% else %}}
+{{% set PACKAGE_NAME = "libsemanage-python" %}}
+{{% endif %}}
+
+- name: Ensure {{{ PACKAGE_NAME }}} installed
+  package:
+    name: "{{{ PACKAGE_NAME }}}"
+    state: present
+
+{{% if product == "sle15" %}}
+- name: "{{{ product }}} - Ensure Additional Packages installed"
```

Comment:
```suggestion
- name: "{{{ rule_title }}} - Ensure Additional Packages installed"
```
`{{{ rule_title }}} -` should be the prefix for all Ansible tasks in a Playbook.
However, there are more rules in the project Style Guide for Ansible with are not satisfied in this Playbook. Could you update it, please?
- https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible